prepare_specialized_urdf.py

```diff
@@ -2,6 +2,7 @@
 import pathlib
 import pprint
 import subprocess
+import sys
```

Comment:
You should rebase on top of `master` to eliminate this commit. (The reason this arises is the default merge strategy for this repo is Squash & Merge)